tasty-plutus/src/Test/Tasty/Plutus/TestData.hs

```diff
@@ -144,8 +145,9 @@ data Generator (p :: Purpose) where
     , FromData redeemer
     , Show redeemer
     ) =>
-    (redeemer -> Example) ->
+    (redeemer -> Value -> Example) ->
```

Comment:
This also requires a `@since` bump, a major version raise, and a CHANGELOG entry, as it changes the public API.